lib/rhc/helpers.rb

```diff
@@ -522,7 +522,13 @@ def to_boolean(s, or_nil=false)
 
     # split spaces but preserve sentences between quotes
     def split_path(s, keep_quotes=false)
-      keep_quotes ? s.split(/\s(?=(?:[^"]|"[^"]*")*$)/) : CSV::parse_line(s, :col_sep => ' ')
+      if keep_quotes
+        s.split(/\s(?=(?:[^"]|"[^"]*")*$)/)
+      elsif RUBY_VERSION.to_f < 1.9
+        CSV::parse_line(s, fs = ' ')
+      else #ruby 1.9 or newer
+        CSV::parse_line(s, :col_sep => ' ')
+      end
```

Comment:
@thrasher-redhat, please see the definition of `with_tolerant_encoding` further down which uses `# :nocov:` markers at the top and bottom of the function definition to override coverage requirements.  We will just need to do the same for `split_path`. @fabianofranz, please let us know if you have a better approach.